src/sdg/producten/management/commands/check_broken_links.py

```diff
@@ -0,0 +1,307 @@
+                        f"online aanvragen ({localized_product.taal})"
+                    ].append((value, url))
+                    self.url_set.add(url)
+
+                elif key in FIELD_NAMES_CONFIG["decentrale_link"]:
+                    label, _ = decentrale_field
+                    if not label:
+                        decentrale_field = (label, value)
+                        continue
+
+                    self.product_dict[product.pk][
+                        f"online aanvragen ({localized_product.taal})"
+                    ].append((label, value))
+                    self.url_set.add(value)
+
+                if not value:
+                    continue
+
+                if key in FIELD_NAMES_CONFIG["markdown_fields"]:
+                    occurring_field = localized_field_verbose_name(key)
+                    html = markdown.markdown(value)
+                    soup = BeautifulSoup(html, "html.parser")
+                    for link in soup.find_all("a"):
+                        text = link.get_text()  # Get the label, request VNG.
+                        href = link["href"]
+                        self.product_dict[product.pk][occurring_field].append(
+                            (text, href)
+                        )
+                        self.url_set.add(href)
+
+                elif key in FIELD_NAMES_CONFIG["urls_fields"]:
+                    occurring_field = localized_field_verbose_name(key)
+                    for label, url in value:
+                        self.product_dict[product.pk][occurring_field].append(
+                            (label, url)
+                        )
+                        self.url_set.add(url)
+
+    def check_url(self, url: str):
+        try:
+            response = self.request_head(url=url)
+            response.close()
+            return (url, response.status_code)
+        except requests.exceptions.InvalidSchema:
+            # These adapters will work most likely in the front-end, but the back-end will not know.
+            if url.startswith(VALID_URL_ADAPTERS):
+                return (url, SUCCES_STATUS_CODE)
+            # Exception for URLs starting with ['tel:', 'sms:', 'geo:', ...]
+            return (url, ANY_ERROR_STATUS_CODE)
+        except requests.RequestException:
+            return (url, ANY_ERROR_STATUS_CODE)
+
+    def handle_response_code(self, url, occurring_field, product_pk, url_label):
+        """
+        Description
+        -----------
+        Increment or delete the broken_link based on the status_code
+
+        Parameters
+        ----------
+        - status_code : int
+            The status_code of the response. (most likely 200, 404 or 420).
+        - url : str
+            The url of the request response.
+        - broken_links : BrokenLinks
+            The broken link class model that interacts with the DB.
+        - url_label : str,
+            The 'label' of the url in the content.
+        """
```

Comment:
This is typically not how we do docstrings.

Although its an opiniated matter, the options are roughly as described here: https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats

And we use reST-style because we mostly use rst-files for documentation.

Reply:
Fixed in https://github.com/maykinmedia/sdg-invoervoorziening/pull/1018/commits/adc06025a452bdece4d943240dc4dc302a80fc53 ✅